src/main/kotlin/org/ionproject/integration/model/internal/timetable/Faculty.kt

```diff
@@ -0,0 +1,7 @@
+package org.ionproject.integration.model.internal.timetable
+
+data class Faculty(
+    var course: String = "",
+    var course_type: String = "",
+    var teachers: List<Teacher> = listOf()
```

Comment:
Same as Course

Reply:
They can't because they aren't filled all at the same time